src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1265,8 +1314,15 @@ rpcClient(std::vector<std::string> const& args,
             if (!setup.client.admin_password.empty ())
                 jvRequest["admin_password"] = setup.client.admin_password;
 
-            jvParams.append (jvRequest);
+            if (!jvRequest.isMember("ripplerpc"))
```

Comment:
I was able to confirm this command line: `--rpc_ip=192.168.42.99 --rpc_port=51235 json2 "{ \"jsonrpc\": \"2.0\", \"ripplerpc\": \"2.0\", \"id\": 5, \"method\": \"account_info\", \"params\": [{ \"account\": \"rrHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh\"}] }"`
resulted in
```
{
   "id" : 5,
   "jsonrpc" : "2.0",
   "result" : {
      "error" : "invalidParams",
      "error_code" : 30,
      "error_message" : "Missing field 'account'.",
      "request" : {
         "command" : "account_info"
      },
      "status" : "error"
   },
   "ripplerpc" : "2.0"
}
```
Assigning `jvParams` regardless gave a valid result.